website/src/documentation/runners/flink.md

```diff
@@ -103,7 +103,7 @@ To find out which version of Flink is compatible with Beam please see the table
   <th>Artifact Id</th>
 </tr>
 <tr>
```

Comment:
We need a new row in this table that specifies the version support from 2.20 (or whatever version we release this change with) going forward.

Reply:
Sure，will update the PR after https://github.com/apache/beam/pull/10945 merged. :)

Reply:
This has been addressed in #10945 and updated in this PR. 2.21.0 will be the first version without 1.7 and with 1.10 support.